media/js/base/stat-counter.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+    'use strict'
```

Comment:
Missing semi-colon here is the reason tests are failing. Will patch manually and merge.